database/artix7/mapping/parts.yaml

```diff
@@ -1,33 +1,440 @@
-# part number to device, package and speed grade mapping
```

Comment:
Now that the mapping and resource files are auto-gen'd, please remove this from the git repo.  Update the CI build script and `README.md` to mention running `make db-prepare-parts`